spec/services/time_zone_builder_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+describe TimeZoneBuilder do
```

Comment:
I would add here: `subject { described_class.new }`
and then instead of `TimeZoneBuilder.new.call(50.0640, 19.9440)` we can have `subject.call(50.0640, 19.9440)`.